content/en/synthetics/guide/uptime-percentage-widget.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Website Uptime monitoring with SLO
+kind: guide
+author: 'Boyan Syarov'
+aliases:
+    - /graphing/guide/uptime-percentage-widget
+    - /dashboards/guide/uptime-percentage-widget
+further_reading:
+    - link: '/monitors/monitor_uptime_widget/'
+      tag: 'Documentation'
+      text: 'Monitor Uptime Widget'
+    - link: '/synthetics/'
+      tag: 'Documentation'
+      text: 'Synthetics'
+---
+
+Maintaining service level agreements with external or internal customers often requires measuring uptime percentage. This guide shows you how to achieve this using Datadog's [Synthetics][1] and the [SLO widget][2]. It uses `http://example.com/` website as an example.
+
+## Synthetics test creation
+
+Create a new [Synthetics API test][3] upon `http://example.com/`:
+
+1. [Go to the new Synthetics API page][4].
+2. Enter `http://example.com/` in the **URL** field.
+3. Click **Test URL** to automatically populate the assertions to monitor:
+
+    {{< img src="synthetics/guide/uptime_slo/synthetics_test_config.png" alt="Synthetics test configuration" >}}
+
+4. Tweak those assertions to match your SLI and define your test retry policy, in this example if at least half of the locations fails, even after one retry, then we consider that the site is down;
+
+    {{< img src="synthetics/guide/uptime_slo/synthetics_test_assertions.png" alt="Synthetics test assertions" >}}
+
+5. Optionally - Define a verbose notification message:
```

Comment:
We now support `{{#is_alert}}`, etc. conditional variables in case you want to include them in the screenshot!